SeaPublicWebsite/Views/StaticPages/PrivacyPolicy.cshtml

```diff
@@ -0,0 +1,184 @@
+            Automated decision making
+        </h2>
+        <p class="govuk-body">
+            Your personal data will not be subject to automated decision making.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            Your Rights
+        </h2>
+        <p class="govuk-body">
+            You have the right to request information about how your personal data are processed, and to request a copy
+            of that personal data.
+        </p>
+        <p class="govuk-body">
+            You have the right to request that any inaccuracies in your personal data are rectified without delay.
+        </p>
+        <p class="govuk-body">
+            You have the right to request that any incomplete personal data are completed, including by means of a
+            supplementary statement.
+        </p>
+        <p class="govuk-body">
+            You have the right to request that your personal data are erased if there is no longer a justification for
+            them to be processed.
+        </p>
+        <p class="govuk-body">
+            You have the right in certain circumstances (for example, where accuracy is contested) to request that the
+            processing of your personal data is restricted.
+        </p>
+        <p class="govuk-body">
+            You have the right to object to the processing of your personal data where it is processed for direct
+            marketing purposes.
+        </p>
+        <p class="govuk-body">
+            You have the right to object to the processing of your personal data.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            International Transfers
+        </h2>
+        <p class="govuk-body">
+            As your personal data is stored on our IT infrastructure and shared with our data processors Microsoft and
+            Amazon Web Services, it may be transferred and stored securely in the UK and European Economic Area. Where
+            it is the case that your personal data is stored outside the UK and EEA it will be subject to equivalent 
+            legal protection through the use of Model Contract Clauses.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            Complaints
+        </h2>
+        <p class="govuk-body">
+            If you consider that your personal data has been misused or mishandled, you may make a complaint to the
+            Information Commissioner, who is an independent regulator.
+        </p>
+        <p class="govuk-body">
+            The Information Commissioner can be contacted at:
+        </p>
+        
+        <div class="govuk-inset-text">
+            <p class="govuk-body">
+                Information Commissioner's Office<br>
+                Wycliffe House<br>
+                Water Lane<br>
+                Wilmslow<br>
+                Cheshire<br>
+                SK9 5AF<br>
+            </p>
+            <ul class="govuk-list">
+                <li>Telephone: 0303 123 1113</li>
```

Comment:
Did we check that this was the right number? I know there was a discussion around that

Reply:
Seems to be: https://ico.org.uk/make-a-complaint/